src/libraries/Native/Unix/System.Globalization.Native/pal_collation.c

```diff
@@ -688,16 +684,16 @@ static int32_t SimpleAffix(const UCollator* pCollator, UErrorCode* pErrorCode, c
         UCollationElements* pSourceIterator = ucol_openElements(pCollator, pText, textLength, pErrorCode);
         if (U_SUCCESS(*pErrorCode))
         {
-            UColAttributeValue strength = ucol_getStrength(pCollator);
+            const UColAttributeValue strength = ucol_getStrength(pCollator);
 
             int32_t capturedOffset = 0;
-            result = SimpleAffix_Iterators(pPatternIterator, pSourceIterator, strength, forwardSearch, (pMatchedLength != NULL) ? &capturedOffset : NULL);
+            result = SimpleAffix_Iterators(pPatternIterator, pSourceIterator, strength, forwardSearch, pMatchedLength != nullptr ? &capturedOffset : nullptr);
```

Comment:
This is still using `nullptr` and causing the build breaks...

Reply:
Ahh ok, I will get this fixed.